zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -495,6 +496,18 @@ object Http {
    */
   def fromOptionFunction[A]: FromOptionFunction[A] = new FromOptionFunction(())
 
+  /**
+   * Creates a Http that always succeeds with a 200 status code and the provided ZStream as the body
+   */
+  def fromStream[R](stream: ZStream[R, Throwable, String], charset: Charset = HTTP_CHARSET): HttpApp[R, Nothing] =
+    Http.fromEffect(ZIO.environment[R].map(r => Http.fromData(HttpData.fromStream(stream.provide(r), charset)))).flatten
+
+  /**
+   * Creates a Http that always succeeds with a 200 status code and the provided ZStream as the body
+   */
+  def fromStream[R](stream: ZStream[R, Throwable, Byte]): HttpApp[R, Nothing] =
+    Http.fromEffect(ZIO.environment[R].map(r => Http.fromData(HttpData.fromStream(stream.provide(r))))).flatten
```

Comment:
I am not sure what will be the impact on Performance here.

Reply:
We will need to benchmark this. Irrespective of that, in production people will use provide anyways at places.